### ISSUE TITLE
deployment in K8S... don't get it.

### ISSUE THREAD

**Issue by strus38** [jumanjihouse/docker-tftp-hpa#31]
Hi
I am trying to use your container inside K8S, but I don't get it.
Here is my deployment file:
`
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: tftp-nfs
  namespace: ftpsvc
  labels:
    app.name: tftpd
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 4Gi
  volumeName: tftp-nfs
  volumeMode: Filesystem

---
apiVersion: v1
kind: Service
metadata:
  name: tftpd
  namespace: ftpsvc
  labels:
    app.name: tftpd
spec:
  type: LoadBalancer
  loadBalancerIP: 192.168.1.199
  ports:
    - port: 69
      targetPort: tftpd
      protocol: UDP
      name: tftpd

---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: tftpd
  namespace: ftpsvc
  labels:
    app.name: tftpd
spec:
  replicas: 1
  selector:
    matchLabels:
      app.name: tftpd
  template:
    metadata:
      labels:
        app.name: tftpd
    spec:
      containers:
        - name: tftpd
          image: jumanjiman/tftp-hpa:latest
          imagePullPolicy: Always
          ports:
            - name: tftpd
              containerPort: 69
              protocol: UDP
          resources:
            limits:
              cpu: 100m
              memory: 128Mi
            requests:
              cpu: 100m
              memory: 128Mi
          volumeMounts:
            - name: tftp-nfs
              mountPath: /tftpboot
              readOnly: false
      volumes:
        - name: tftp-nfs
          persistentVolumeClaim:
            claimName: tftp-nfs
`
So basically, trying to mount in /tftpboot on my NFS mount.
Did you ever try it in K8S?
Do you have a sample deployment file?
Thanks
